RELEASING.md

```diff
@@ -42,20 +32,21 @@ ossrhUsername=<YOUR-NEXUS-USERNAME>
 ossrhPassword=<YOUR-NEXUS-PASSWORD>
 ```
 
-Deploy to Sonatype
-------------------
-* Update all ```pom.xml``` files in the package to the release version you want. Sumit a pull request, get it reviewed, and submit
-* ```mvn clean install deploy -DperformRelease=true```
-* Verify the publishment [here](https://oss.sonatype.org/#nexus-search;quick~gax)
-  * If there is a problem, undo by ```mvn nexus-staging:drop```
-* ```mvn nexus-staging:release -DperformRelease=true```
-* Update all ```pom.xml``` files to the new snapshot version
+Update version and deploy to Sonatype
+-------------------------------------
+* Update the ```gax/version.txt``` and ```gax-grpc/version.txt``` files to the
+  release version you want
+* Run ```./gradlew stageRelease``` to:
+  * Update ```README.md``` and ```samples/pom.xml```
+  * Regenerate ```gh-pages``` branch containing Javadocs
+  * Stage artifacts on Sonatype: to the staging repository if "-SNAPSHOT" is *not* included in the version; otherwise to the snapshot repository only
+* Submit a pull request, get it reviewed, and submit
 
 Publish the release
 -------------------
-* Go to [Sonatype](https://oss.sonatype.org/) and log in
-* Click on *Staging Repositories* on the left
-* Filter down to the repository by typing the package's groupId without periods in the search box
-  * In our case, ```comgoogleapi```
-* Click the *close*, then *release* button just below the top tabs
+* Run ```./gradlew finalizeRelease```
 * It will take some time (up to 10 minutes) for the package to transition
+
+Bump development version
+------------------------
+* Repeat the section "Update version and deploy to Sonatype" above, updating versions to the following "-SNAPSHOT" version
```

Comment:
Actually, the only thing that should be done is bumping the versions in the version.txt files.